WindowsServerDocs/administration/OpenSSH/OpenSSH_KeyManagement.md

```diff
@@ -142,19 +152,14 @@ After completing these steps, whenever a private key is needed for authenticatio
 To use the user key that was created above, the public key needs to be placed on the server into a text file called *authorized_keys* under users\username\\.ssh\\.
 The OpenSSH tools include scp, which is a secure file-transfer utility, to help with this.
 
-To move the contents of your public key (~\.ssh\id_ed25519.pub) into a text file called authorized_keys in ~\.ssh\ on your server/host.
-
-This example uses the Repair-AuthorizedKeyPermissions function in the OpenSSHUtils module which was previously installed on the host in the instructions above.
+To move the contents of your public key (~\.ssh\id_ed25519.pub) into a text file called authorized_keys in ~\.ssh\ on your server.
```

Comment:
Is [`ssh-copy-id`](https://www.ssh.com/ssh/copy-id) still not part of the Win32 port of OpenSSH?

Reply:
Not on my Windows 10 installation at least.